src/p2p/net_node.inl

```diff
@@ -1841,7 +1841,7 @@ namespace nodetool
   }
   //-----------------------------------------------------------------------------------
   template<class t_payload_net_handler>
-  bool node_server<t_payload_net_handler>::sanitize_peerlist(std::vector<peerlist_entry>& local_peerlist)
+  bool node_server<t_payload_net_handler>::sanitize_peerlist(const epee::net_utils::connection_context_base& context, std::vector<peerlist_entry>& local_peerlist)
```

Comment:
Why do you add `context` parameter?

Reply:
Ah, that was for the log I had. I'll remove it.